lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -134,6 +134,7 @@ def create_release(repository:, version:, description:, assets:, prerelease:, is
         assets.each do |file_path|
           client.upload_asset(release[:url], file_path, content_type: 'application/octet-stream')
         end
+        release[:url]
```

Comment:
I just realized [looking at the API docs](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release) that the `url` field in the response looks like `https://api.github.com/repos/octocat/Hello-World/releases/1`, while what we want to return and print is the `html_url` field, which looks like `https://github.com/octocat/Hello-World/releases/v1.0.0`

```suggestion
        release[:html_url]
```

Reply:
Fixed in 0128b8dc

Reply:
Aahh.. Funnily enough, I actually knew about this, but until you pointed out, completely forgot 🤦 Thanks for catching that before the PR lands.